src/util/scoring/css.rs

```diff
@@ -52,6 +54,39 @@ async fn render_image(code: &str, width: u32, height: u32) -> anyhow::Result<Vec
     Ok(image_buffer)
 }
 
+#[tracing::instrument(err)]
+pub async fn render_diff_image(
+    question_image_buffer: &[u8],
+    html: String,
+) -> anyhow::Result<(f32, Vec<u8>)> {
+    let question_image =
+        image::load_from_memory_with_format(question_image_buffer, ImageFormat::Png)?;
+    let (width, height) = question_image.dimensions();
+    let answer_image_buffer = render_image(&html, width, height).await?;
+
+    let mut diff_image_buffer = Vec::with_capacity((height * width) as usize);
+
+    match pixelmatch(
```

Comment:
use "?" pattern instead

Reply:
![image](https://github.com/F-Code-Club/rode-be-socket-f-code.tech/assets/124784766/47bc0712-82b5-4b44-a007-e0c07b8316ec)
Did you mean this?
But this code has an error:
![image](https://github.com/F-Code-Club/rode-be-socket-f-code.tech/assets/124784766/782561fb-0e39-4602-9d74-68b8c8cd6d44)